common/src/service/HttpClientHelper.cs

```diff
@@ -674,8 +686,12 @@ private static void InsertEtag(HttpRequestMessage requestMessage, string etag)
             IDictionary<HttpStatusCode, Func<HttpResponseMessage, Task<Exception>>> errorMappingOverrides,
             CancellationToken cancellationToken)
         {
-            using (var httpClient = BuildHttpClient(_defaultOperationTimeout))
+            HttpClient httpClient = null;
+            HttpClientHandler httpClientHandler = null;
+
+            try
```

Comment:
I should have probably pointed this out too :) Same suggestion as above for using.